src/GeneratedDancer.js

```diff
@@ -0,0 +1,99 @@
+// Thin p5 adapter: owns a p5.Graphics mid-layer and gives its 2D context
+// to the external renderer. CommonJS to match the rest of dance-party.
+
+// Map of move IDs to move names.
+const movesById = {
```

Comment:
Minor: share/consolidate with the list of names in https://github.com/code-dot-org/dance-party/blob/main/src/constants.js?